detekt.yml

```diff
@@ -7,6 +7,7 @@ naming:
 
 complexity:
     LongParameterList:
+        functionThreshold: 8
```

Comment:
I would prefer to suppress warnings on specific functions when necessary rather than changing the global default. Ideally non-Composable functions can be kept to 6.

Reply:
I agree, but there are functions that are not Composable functions, but at the same time are part of the implementation of Composable functions - this often requires a large number of parameters. I corrected this in favor of annotating the necessary methods.